elado/pannonhalma/index.md

```diff
@@ -0,0 +1,121 @@
+
+layout: elado
+title:  Vadregényes vízi-világ eladó Pannonhalmán 
+
+
+**Irányára: 3500000 €**
+---
+
+#![Kívülről](http://i.imgur.com/J4m9gPe.jpg) Vadregényes vízi-világ eladó Pannonhalmán
+
+{% include fold.html %}
+
+Eladó Pannonhalmán egy 80 ha-os, vadregényes ingatlan. 
+
+A birtokot a Világörökség részét képező Pannonhalmi Bencés Apátság impozáns épület-együttese őrzi. A tulajdonos család szorgalmát, igényességét dicséri a hatalmas, mindenhol gondozott terület.
+
+![Kívülről](http://i.imgur.com/gxLCTPr.jpg)
```

Comment:
OK. Köszönöm.
Valamikor az elején ezt így csináltuk, akkor véstem be és így maradtam...
pedig már mondtad is. :)

Sajna nem látja az új hirdetést a program... Ha rámegyek a "részletek"-re,
404-es hibát jelez!
Valamit biztosan elszúrtam... Bocsánat!

Örülök, hogy megérkeztetek! :)